packages/core/mesh/edge-client/src/client.ts

```diff
@@ -18,17 +19,13 @@ const DEFAULT_TIMEOUT = 5_000;
 
 export type MessageListener = (message: Message) => void | Promise<void>;
 
-/**
- *
- */
-export interface EdgeConnection {
+export interface EdgeConnection extends Required<Lifecycle> {
   get info(): any;
   get identityKey(): PublicKey;
   get deviceKey(): PublicKey;
   get isOpen(): boolean;
+  setIdentity({ deviceKey, identityKey }: { deviceKey: PublicKey; identityKey: PublicKey }): void;
```

Comment:
are you planning to change those to strings?

Reply:
Yes